specs/windows/windows_firewall_rules.table

```diff
@@ -0,0 +1,22 @@
+table_name("windows_firewall_rules")
+description("Provides the list of Windows firewall rules.")
+schema([
+    Column("name", TEXT, "Friendly name of the rule"),
+    Column("app_name", TEXT, "Friendly name of the application to which the rule applies"),
```

Comment:
Is there a non-friendly, but authoritative, way to link this rule to an app? (I haven't looked at the underlying API yet)

Reply:
Digging through https://docs.microsoft.com/en-us/windows/win32/api/netfw/nn-netfw-inetfwrule

I think `get_ServiceName` is worth adding, since that's documented as an optional parameter that may take the place of app name.

`get_grouping` might also be interesting

Reply:
basically exposed this API: (https://docs.microsoft.com/en-us/windows/win32/api/netfw/nn-netfw-inetfwrule)
and got the description from https://docs.microsoft.com/en-us/windows/win32/api/netfw/nf-netfw-inetfwrule-get_applicationname

Reply:
will add